src/Database/InfluxDB/Write.hs

```diff
@@ -139,6 +142,12 @@ writeRequest WriteParams {..} =
       , do
         Key name <- _retentionPolicy
         return ("rp", Just (TE.encodeUtf8 name))
+      , do
+        Credentials { _user = u } <- _authentication
+        return ("u", Just (TE.encodeUtf8 u))
+      , do
+        Credentials { _password = p } <- _authentication
+        return ("p", Just (TE.encodeUtf8 p))
```

Comment:
Can we change this list of type `[Maybe (ByteString, Maybe ByteString)]` into `[[(ByteString, Maybe ByteString)]]` so that we can combine these "u" and "p" into one `do` block?